cmd/podman/stats.go

```diff
@@ -84,8 +84,7 @@ func statsCmd(c *cli.Context) error {
 	if ctr > 1 {
 		return errors.Errorf("--all, --latest and containers cannot be used together")
 	} else if ctr == 0 {
-		// If user didn't specify, imply --all
-		all = true
+		return errors.Errorf("you must specify --all, --latest, or at least one container")
```

Comment:
Why not stick with all as the default?  That's what I'd expect if I didn't specify anything.

Reply:
because @rhatdan complained

Reply:
@TomSweeneyRedHat If we are going to do that, then why have --all?

This is also how podman top works.
```
# podman top 
you must provide the name or id of a running container
```

Reply:
Having --all is nice for OCD folks like myself who want to be specific at times.  

I think top and stat should behave in the same manner, I'd prefer default -all for both, but not a biggy.